sycl/test-e2e/Basic/context_platforms.cpp

```diff
@@ -1,5 +1,6 @@
-// RUN: %clangxx -fsycl %s -o %t.out
-// RUN: %BE_RUN_PLACEHOLDER %t.out
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
```

Comment:
This should probably be kept as a BE_RUN_PLACEHOLDER, since the test relies on having multiple platforms.